README.md

```diff
@@ -65,15 +71,17 @@ make run
 ## Usage and Examples
 Create a new healthcheck:
 
-`kubectl create -f https://raw.githubusercontent.com/keikoproj/active-monitor/master/examples/inlineHello.yaml`
+`kubectl create ns test`
```

Comment:
how about if we provide this as a new example rather than overwriting the existing one

Reply:
sure will update with the changes